L1Trigger/L1TTwinMux/src/L1MuTMChambPhContainer.cc

```diff
@@ -0,0 +1,128 @@
+//-------------------------------------------------
+//
+//   Class L1MuTMChambPhContainer
+//
+//   
+//
+//
+//   Author List: Jorge Troconiz, George Karathanasis
+//
+//
+//--------------------------------------------------
+
+//-----------------------
+// This Class's Header --
+//-----------------------
+#include "L1Trigger/L1TTwinMux/interface/L1MuTMChambPhContainer.h"
+
+//-------------------------------
+// Collaborating Class Headers --
+//-------------------------------
+
+
+//---------------
+// C++ Headers --
+//---------------
+using namespace std;
+
+//-------------------
+// Initializations --
+//-------------------
+
+
+//----------------
+// Constructors --
+//----------------
+L1MuTMChambPhContainer::L1MuTMChambPhContainer() {}
+
+//--------------
+// Destructor --
+//--------------
+L1MuTMChambPhContainer::~L1MuTMChambPhContainer() {}
+
+//--------------
+// Operations --
+//--------------
+void L1MuTMChambPhContainer::setContainer(const Phi_Container& inputSegments) {
+
+  phiSegments = inputSegments;
+}
+
+L1MuTMChambPhContainer::Phi_Container const* L1MuTMChambPhContainer::getContainer() const {
+  return &phiSegments;
+}
+
+bool L1MuTMChambPhContainer::bxEmpty(int step) const {
+
+  bool empty = true;
+
+  for ( Phi_iterator i  = phiSegments.begin();
+                     i != phiSegments.end();
+                     i++ ) {
+    if  (step == i->bxNum()) empty = false;
```

Comment:
this if statement can just return  (and then you don't need the empty variable at all)